src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file was `Vector.cs`

Reply:
I removed the regions and simplified the code, but otherwise left the general ordering of the file the same as it already was.